doc/api/test.md

```diff
@@ -202,7 +202,8 @@ If Node.js is started with the [`--test-only`][] command-line option, it is
 possible to skip all top level tests except for a selected subset by passing
 the `only` option to the tests that should be run. When a test with the `only`
 option set is run, all subtests are also run. The test context's `runOnly()`
-method can be used to implement the same behavior at the subtest level.
+method can be used to implement the same behavior at the subtest level. Tests
+that are not executed are omitted from the test runner output.
```

Comment:
Skip tests are not executed but do appear, don't they?

Reply:
That's correct (precedence for behavior is filtering > skip > todo). I wasn't sure on the best wording to describe that in these two paragraphs.